attendance/lib/ProfilePage/profile_page_view.dart

```diff
@@ -0,0 +1,198 @@
+import 'package:flutter/material.dart';
+
+import './profile_page_viewmodel.dart';
+
+class ProfilePageView extends ProfilePageViewModel {
+  @override
+  Widget build(BuildContext context) {
+    return new Scaffold(
```

Comment:
@Bahaaib can you add an app bar? currently no up button

Reply:
The App bar here will blow up the page design

Reply:
@Bahaaib the up button is necessary for all pages (except login and main page), that's a UX rule (we have iOS users who will kill us)